src/style.css

```diff
@@ -1 +1,37 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  height: 100px;
+  width: 100px;
+}
```

Comment:
Don't fix container size ([checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md#css)). delete style justify-content and define gaps between stars using margin